src/main/resources/org/jenkinsci/plugins/envinject/EnvInjectBuildWrapper/help-propertiesContent.html

```diff
@@ -0,0 +1,6 @@
+<div>
```

Comment:
@oleg-nenashev I don't know what you mean with *"...reference a text with extra details by URL."*